system/src/system_network_internal.cpp

```diff
@@ -17,6 +17,62 @@
 
 #include "system_network_internal.h"
 
+namespace particle {
+
+namespace {
+
+bool networkOff(network_interface_index iface) {
```

Comment:
`newhal` platforms seems to fail building, these should probably get wrapped in `SPARK_NO_CLOUD` or something like that.

Reply:
:+1: